.github/ISSUE_TEMPALTE/question.md

```diff
@@ -0,0 +1,11 @@
+---
+name: ❔Question
+about: Ask a question about the package or any of its features.
+title: ""
+labels: "question"
+assignees: ""
+---
+
+### Question
+
+<!-- Please make sure to provide as much context as possible so we can help you better. -->
```

Comment:
You raise a very good point! I removed the template and added a contact_link in the `config.yml`.
But I linked the GH Discussions Tab instead of the community, as this is the way I will respond the quickest.

If you ever stumble across something PHP related in the community, feel free to give me a heads up or direct the person here.